src/SVMTrainer.js

```diff
@@ -14,20 +20,59 @@ export default class SVMTrainer {
     this.svm = new svmjs.SVM();
     const state = store.getState();
     this.state = state;
+    this.buildFeatureToNumberKey(state);
   }
 
   startTraining() {
     this.addTrainingData();
   }
 
+  buildFeatureToNumberKeyForColumn = column => {
```

Comment:
Could we get a small comment for each function?  And maybe one bigger comment somewhere documenting how this new feature ties together?

Reply:
I added comments and fleshed out the description in the PR. Anything missing that would help?